src/gui_tool/gui_managers.py

```diff
@@ -142,6 +165,8 @@ def play_video(self):
             mark_changed = False
             received_key = cv2.waitKey(self.frame_rate) & 0xFF
             if received_key == 27:  # Escape key
+                raise ManualTaggingAbortedException("Tagging operation aborted")
+            if received_key == 13:  # Enter
```

Comment:
Might be worth creating an enum or extending the ord() function to capture these keys since they're used in multiple places in the code